docs/courses/csintro3/functions/comments.md

```diff
@@ -1,9 +1,213 @@
-# Comments for Functions
+# Activity: Function Documentation
 
-give description of standard function commenting
+In this section, ``||functions:functions||`` have been shown to have a wide variety of uses, with many different features.
```

Comment:
"In this section" sounds like you're referring to the current page instead of the previous activities

Reply:
Yeah i was thinking on that, any alt wording? We don't really have a consistent way of referring to the "chapters" as it hasn't come up much